cal/cal/templates/home_logged_in.html

```diff
@@ -85,11 +85,19 @@ <h4 class="well">Categories - Last Week</h4>
       <h4 class="well">Tags - Cumulative</h4>
         <table class="table table-hover">
           <thead>
-            <th>Label</th>
-            <th>Hours</th>
+            <th>
+              <a href="" ng-click="orderByField = 'label'; reverseSort = !reverseSort">
+                Label
+              </a>
+            </th>
+            <th>
+              <a href="" ng-click="orderByField = 'hours'; reverseSort = !reverseSort">
+                Hours
+              </a>
+            </th>
           </thead>
           <tbody>
-            <tr ng-repeat="tag in tags">
+            <tr ng-repeat="tag in tags | orderBy:orderByField:reverseSort">
```

Comment:
Merge from master and it should go away. Your branch is currently out of date so Github thinks you want to undo Tiffany's commits